spec/index.bs

```diff
@@ -1179,10 +1177,19 @@ partial interface HTMLScriptElement {
 
 This document modifies {{HTMLScriptElement}}s. Each script has:
 
-: an associated string <dfn export for="HTMLScriptElement">script text</dfn>.
-::  A string, containing the body of the script to execute that was set
-    through a compliant sink. Equivalent to script's
-    [=child text content=]. Initially an empty string.
+: an associated boolean <dfn export for="HTMLScriptElement">is trusted</dfn>.
+::  A boolean indicating whether a script element is considered trustworthy for execution.
+    Initially true.
```

Comment:
Initially true which covers scripts inline in the page being parsed.

Reply:
Might be worth promoting that comment to a note in the spec.